packages/marketplace/src/components/ui/developer-analytics/cost-explorer/service-chart/service-chart.tsx

```diff
@@ -59,18 +82,32 @@ export const mapServiceChartDataSet = (billing: BillingOverviewForPeriodV2Model
     const apiCallsData = services.find(service => service.name === 'API Requests')?.cost || 0
     const developerEditionData = services.find(service => service.name === 'Developer Edition')?.cost || 0
     const appListingData = services.find(service => service.name === 'Application Listing')?.cost || 0
+    const reapitConnectData = services.find(service => service.name === 'Reapit Connect')?.cost || 0
+    const developerRegistrationData = services.find(service => service.name === 'Developer Registration')?.cost || 0
+
+    clonedDataSet[0].totalCost += apiCallsData
+    clonedDataSet[1].totalCost += appListingData
+    clonedDataSet[2].totalCost += developerEditionData
+    clonedDataSet[3].totalCost += reapitConnectData
+    clonedDataSet[4].totalCost += developerRegistrationData
 
     // api calls
     clonedDataSet[0].data.push(apiCallsData)
     // app listing
     clonedDataSet[1].data.push(appListingData)
     // developer edition
     clonedDataSet[2].data.push(developerEditionData)
+    //Reapit Connect
+    clonedDataSet[3].data.push(reapitConnectData)
+    //Developer Registration'
+    clonedDataSet[4].data.push(developerRegistrationData)
```

Comment:
can we use
```
const DATA_INDEXES = {
  API_CALL: 0,
  APP_LISTING: 0,
  DEVELOPER_EDITION: 0,
  DEVELOPER_REGISTRATION: 0,  
}
```
and get rid of the comment

Reply:
Or something like `Record<DATA (which is 'API_CALL' | ... ), type of dataSetItem)`

Reply:
@vuhuucuong @phmngocnghia Thanks. I updated the indexes. It more clearly